eng/pipelines/templates/jobs/archetype-sdk-integration.yml

```diff
@@ -146,11 +208,13 @@ jobs:
           npm install
         workingDirectory: $(Build.SourcesDirectory)/eng/tools/eng-package-utils
         displayName: "Install script dependencies"
+        condition: eq(variables['DependencyVersion'],'repo')
```

Comment:
To help clarify these steps I would combine them into one and use the `eq(variables['PublishCodeCoverage'], true)` as that is what they are really needed for.